package.json

```diff
@@ -46,7 +46,7 @@
     "csv": "0.3.7",
     "debug": "2.2.0",
     "optimist": "0.6.1",
-    "sphere-node-sdk": "1.7.0",
+    "sphere-node-sdk": "1.14.0",
```

Comment:
@lojzatran Please change it to: ^1.14.0 so that it matches in the future all feature versions automatically